fe/fe-core/src/main/java/org/apache/doris/cloud/datasource/CloudInternalCatalog.java

```diff
@@ -80,7 +80,7 @@ public CloudInternalCatalog() {
     @Override
     protected Partition createPartitionWithIndices(long dbId, long tableId, String tableName,
             long baseIndexId, long partitionId, String partitionName, Map<Long, MaterializedIndexMeta> indexIdToMeta,
-            DistributionInfo distributionInfo, TStorageMedium storageMedium, ReplicaAllocation replicaAlloc,
+            DistributionInfo distributionInfo, DataProperty dataProperty, ReplicaAllocation replicaAlloc,
```

Comment:
can delete line 89's arg 'isStorageMediumSpecified'.  Inside createPartitionWithIndices,  it can get with dataProperty.isStorageMediumSpecified()

Reply:
fixed